darts/utils/historical_forecasts/optimized_historical_forecasts_torch.py

```diff
@@ -24,16 +26,15 @@ def _optimized_historical_forecasts(
     series: Sequence[TimeSeries],
     past_covariates: Optional[Sequence[TimeSeries]] = None,
     future_covariates: Optional[Sequence[TimeSeries]] = None,
-    num_samples: int = 1,
     start: Optional[Union[pd.Timestamp, float, int]] = None,
     start_format: Literal["position", "value"] = "value",
     forecast_horizon: int = 1,
     stride: int = 1,
     overlap_end: bool = False,
     last_points_only: bool = True,
     show_warnings: bool = True,
-    predict_likelihood_parameters: bool = False,
     verbose: bool = False,
+    **kwargs,
```

Comment:
same comment with function signature as above for `num_samples` and `predict_likelihood_parameters`